src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer" + " started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer" + " finished work");
+    }
```

Comment:
```suggestion
    public void doWork() {
        System.out.println("Bulldozer started work");
    }

    @Override
    public void stopWork() {
        System.out.println("Bulldozer finished work");
    }
```

Why do we need to split this message into 2 parts? Please, fix in all places